### PR TITLE
update dependencies of sample apps to fix security vulnerability

### PR DESCRIPTION
**Pull request by johnsonshih**
**What this PR does / why we need it**:
Update the version of sample application dependencies to fix security vulnerability.
https://github.com/project-akri/akri/security/dependabot/22
https://github.com/project-akri/akri/security/dependabot/23
https://github.com/project-akri/akri/security/dependabot/41
https://github.com/project-akri/akri/security/dependabot/42
https://github.com/project-akri/akri/security/dependabot/50
https://github.com/project-akri/akri/security/dependabot/51

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR has an associated PR with documentation in [akri-docs](https://github.com/project-akri/akri-docs)
- [ ] this PR contains unit tests
- [ ] added code adheres to standard Rust formatting (`cargo fmt`)
- [ ] code builds properly (`cargo build`)
- [ ] code is free of common mistakes (`cargo clippy`)
- [ ] all Akri tests succeed (`cargo test`)
- [ ] inline documentation builds (`cargo doc`)
- [ ] all commits pass the [DCO bot check](https://probot.github.io/apps/dco/) by being signed off -- see the failing DCO check for instructions on how to retroactively sign commits

### CODE REVIEW

**Review by diconico07 — APPROVED** [review]
Clearly nice to upgrade, but, are those versions tested ?
And as I just see that our image builds don't use these versions (they use the ones from debian package manager), does it matter at all ?

**Comment by johnsonshih** [comment]
> @johnsonshih which CVE is this for? can you add the issue with `fixes`?

add the security alerts in PR description

**Comment by johnsonshih** [comment]
> Clearly nice to upgrade, but, are those versions tested ? And as I just see that our image builds don't use these versions (they use the ones from debian package manager), does it matter at all ?

I had updated the build to pick up specified version for dependencies, both sample app tested.

File: build/containers/Dockerfile.video-streaming-app [hunk 1]
```diff
@@ -4,14 +4,14 @@ FROM ${PLATFORM}/debian:bullseye-slim
 # Avoid tzdata prompt 
 ARG DEBIAN_FRONTEND=noninteractive
 
+WORKDIR /app
+COPY ./samples/apps/video-streaming-app .
+
 RUN echo "Creating container based on ${PLATFORM}/debian:bullseye-slim" && \
     apt-get update && \
     apt-get install -y protobuf-compiler libprotoc-dev python3-pip python3-grpcio python3-kubernetes && \
```
Comment (diconico07) [inline build/containers/Dockerfile.video-streaming-app:12]: maybe remove `python3-grpcio` and `python3-kubernetes` from here, as they'll get installed via pip later

File: build/containers/Dockerfile.anomaly-detection-app [hunk 2]
```diff
@@ -4,15 +4,15 @@ FROM ${PLATFORM}/debian:bullseye-slim
 # Avoid tzdata prompt 
 ARG DEBIAN_FRONTEND=noninteractive
 
+WORKDIR /app
+COPY ./samples/apps/anomaly-detection-app .
+
 RUN echo "Creating container based on ${PLATFORM}/debian:bullseye-slim" && \
     apt-get update && \
     apt-get install -y protobuf-compiler libprotoc-dev python3-pip \
     python3-grpcio python3-sklearn && \
```
Comment (diconico07) [inline build/containers/Dockerfile.anomaly-detection-app:13]: Same for `python3-grpcio` and `python3-sklearn`

Comment (johnsonshih) [reply build/containers/Dockerfile.anomaly-detection-app]: remove

**Review by diconico07 — APPROVED** [review]

**Pull request merged**